src/lightning/fabric/plugins/precision/fsdp.py

```diff
@@ -153,8 +153,3 @@ def state_dict(self) -> Dict[str, Any]:
     def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
         if self.scaler is not None:
             self.scaler.load_state_dict(state_dict)
-
-    def _autocast_context_manager(self) -> torch.autocast:
-        # the dtype could be automatically inferred but we need to manually set it due to a bug upstream
-        # https://github.com/pytorch/pytorch/issues/67233
```

Comment:
the workaround is no longer needed and the comment outdated. I will clean up the rest of occurrences in Lightning in #18634